common/test/acceptance/tests/video/test_studio_video_module.py

```diff
@@ -320,5 +320,3 @@ def test_slider_range_works(self):
         self.save_unit_settings()
 
         self.video.click_player_button('play')
-
-        self.assertTrue(self.video.is_slider_range_visible)
```

Comment:
The current implementation of start/stop times uses a jquery ui slider range to show the instructor-specified start/stop times. This PR removes the slider range and enforces hard start / stop times, so this test should no longer be checking for the slider range's visibility.